apps/src/templates/instructions/MarkdownInstructions.jsx

```diff
@@ -27,15 +26,15 @@ const styles = {
 class MarkdownInstructions extends React.Component {
   static propTypes = {
     markdown: PropTypes.string.isRequired,
-    noInstructionsWhenCollapsed: PropTypes.bool,
+    collapsible: PropTypes.bool,
     onResize: PropTypes.func,
     inTopPane: PropTypes.bool,
     isBlockly: PropTypes.bool,
     showImageDialog: PropTypes.func
   };
 
   static defaultProps = {
-    noInstructionsWhenCollapsed: false
+    collapsible: false
```

Comment:
nit: I don't know if collapsible fully captures where noInstructionsWhenCollapsed(which is a bad name anyway) is supposed to capture. Both CSF and CSD/CSP are collapsible. Jus with CSD/CSP you can collapse to the point of no instructions where as CSF collapses between short and long instructions.  Also are we really just using it here to say CSF vs CSD/CSP and if so maybe we should just change it to that?

Reply:
For this component specifically, this prop is copied into a variable called `canCollapse` so `collapsible` seems like a reasonable prop name. However, I think that's true that `collapsible` doesn't fully capture what `noInstructionsWhenCollapsed` covers in other components.

I'll do a pass of the props I changed/added and see if I can remove/clarify any of them.